x-pack/plugins/session_view/public/components/tty_player_controls/index.tsx

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useCallback, ChangeEvent, MouseEvent, useMemo } from 'react';
+import {
+  EuiPanel,
+  EuiRange,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonIcon,
+  EuiToolTip,
+} from '@elastic/eui';
+import { ProcessEntityIdIOLine } from '../../../common/types/process_tree';
+import { useStyles } from './styles';
+import { TTY_END, TTY_NEXT, TTY_PAUSE, TTY_PLAY, TTY_PREVIOUS, TTY_START } from './translations';
+
+export interface TTYPlayerControlsDeps {
+  currentProcessEntityId: string | undefined;
+  processIdLineMap: Record<string, ProcessEntityIdIOLine>;
+  lastProcessEntityId: string | undefined;
+  isPlaying: boolean;
+  currentLine: number;
+  linesLength: number;
+  onSeekLine(line: number): void;
+  onTogglePlayback(): void;
+  textSizer: JSX.Element;
+}
+
+export const TTYPlayerControls = ({
+  currentProcessEntityId,
+  processIdLineMap,
+  lastProcessEntityId,
+  isPlaying,
+  currentLine,
+  linesLength,
+  onSeekLine,
+  onTogglePlayback,
+  textSizer,
+}: TTYPlayerControlsDeps) => {
+  const styles = useStyles();
+
+  const isFirstProcess = useMemo(
+    () =>
+      (currentProcessEntityId && processIdLineMap[currentProcessEntityId].previous === undefined) ||
+      false,
+    [currentProcessEntityId, processIdLineMap]
+  );
+  const isLastProcess = useMemo(
+    () =>
+      (currentProcessEntityId && processIdLineMap[currentProcessEntityId].next === undefined) ||
+      false,
+    [currentProcessEntityId, processIdLineMap]
+  );
+
+  const onLineChange = useCallback(
+    (event: ChangeEvent<HTMLInputElement> | MouseEvent<HTMLButtonElement>) => {
+      const line = parseInt((event?.target as HTMLInputElement).value || '0', 10);
+      onSeekLine(line);
+    },
+    [onSeekLine]
+  );
+
+  const seekToStart = useCallback(() => {
+    onSeekLine(0);
+  }, [onSeekLine]);
+
+  const seekToEnd = useCallback(() => {
+    if (lastProcessEntityId) {
+      onSeekLine(processIdLineMap[lastProcessEntityId]?.value);
```

Comment:
I think we might just want to seek to the very end of the output, not the start of the last process. Though, I'd check with Max to be sure.